docs/website/content/v0.7/en/guides/local/registry-cache.md

```diff
@@ -81,6 +81,26 @@ talosctl cluster create --provisioner docker \
     --registry-mirror gcr.io=http://172.17.0.1:5003
 ```
 
+## Adding Talos Images to the Registries
```

Comment:
Question about this. Is it only specific to you version of talosctl you've got locally? If so, might be worth noting that.

Reply:
This doesn't work with caching registries in this guide (they're read-only). We need probably a different guide on how to push to a non-caching registry.